src/Workspaces/Core/Portable/Formatting/Engine/AbstractFormatEngine.OperationApplier.cs

```diff
@@ -185,14 +185,31 @@ private bool ApplyForceLinesOperation(AdjustNewLinesOperation operation, int pai
                 return true;
             }
 
+            private int getIndentation(int pairIndex, TriviaData triviaInfo)
```

Comment:
@basoundr I dont believe this is right way to do it. format engine should not know about smart indentation.

rather than this, change smart indenter's specific formatting rules to return null for all indentation operation. and make NewLineOperation to always return null. that will make no new line to happen.